homeassistant/components/plugwise/const.py

```diff
@@ -25,6 +25,7 @@
 PLATFORMS_GATEWAY: Final[list[str]] = [
     Platform.BINARY_SENSOR,
     Platform.CLIMATE,
+    Platform.NUMBER,
```

Comment:
Not from this PR, but SENSOR and SELECT don't follow alphabetical order 😄

Reply:
Yes, will fix this later, NL-evening.